format/FlightSql.proto

```diff
@@ -967,8 +967,8 @@ enum Searchable {
   SEARCHABLE_CHAR = 1;
 
   /**
-   * Indicates that the column can be used in a WHERE clause using other predicates
-   * except for LIKE.
+   * Indicates that the column can be used in a WHERE clause using any other predicates,
+   * except for the LIKE one.
```

Comment:
This is still a little unclear, maybe "…in a WHERE clause with any operator other than LIKE"? (Since SEARCHABLE_FULL uses 'operator' instead of 'predicate')

Reply:
Thanks David. I will be using your suggestion. It's better now